app/components/view-goal/view-goal.jsx

```diff
@@ -1,8 +1,48 @@
 import React from 'react';
+import GoalTileComponent from '../goal-tile.jsx'
+import ProgressBarComponent from './progress-bar.jsx'
+
+const viewGoal = ( { currentGoal } ) => {
+  const rateGoal = (goal) => {
+    console.log("hello");
+  };
+  const rating = [5, 6, 7];
+  const progressBars = rating.map( rating =>
+     <ProgressBarComponent
+       key={rating}
+       rating={rating}
```

Comment:
each day has different progress rating, that's why we have different progress bars